src/service/performance-impl.js

```diff
@@ -420,8 +441,7 @@ export class Performance {
         );
         navigationObserver.observe({type: 'navigation', buffered: true});
       } catch (err) {
-        dev() /*OK*/
-          .error(err);
+        dev()./*OK*/ error(err);
```

Comment:
The warning here is that you're supposed to passing a `TAG`: `dev().error(TAG, err)`

Reply:
done. updated the error report in the moved `try/catch`